librz/core/canalysis.c

```diff
@@ -6809,10 +6814,6 @@ RZ_API void rz_analysis_bytes_free(RZ_NULLABLE void *ptr) {
  */
 RZ_API RZ_OWN RzPVector *rz_core_analysis_bytes(RZ_NONNULL RzCore *core, RZ_NONNULL const ut8 *buf, int len, int nops) {
```

Comment:
Do you know why `cmd_analysis.c:core_analysis_bytes_json()` uses this function but `core_analysis_bytes_json:core_analysis_bytes_standard()` does not? Would it make sense to use this function also there to collect the information about the analyzed bytes and iterate over the result vec?